cypress.config.ts

```diff
@@ -38,6 +38,6 @@ export default defineConfig({
     baseUrl: 'http://localhost:3007',
     excludeSpecPattern: '**/!(*.cy).ts',
     specPattern: 'integration_tests/e2e/**/*.cy.{js,jsx,ts,tsx}',
-    supportFile: 'integration_tests/support/index.ts',
+    supportFile: 'integration_tests/support/createIncident.ts',
```

Comment:
you don’t need to change this line; it’s causing integration tests to fail and we’re not yet making any new ones

Reply:
I don't remember changing this, not sure if it somehow changes when I refactored something.
Thanks as I was going to ask how to fix this